### PR TITLE
Added an .gitattributes file.

### PR DESCRIPTION
**Pull request by cafferata**
Added an .gitattributes file.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/.idea export-ignore
```
Comment (dnl-blkv) [inline .gitattributes]: Don't we want to have `composer.lock` here?
How about `test` directory?

Comment (cafferata) [inline .gitattributes]: This could indeed. I have added these to the `.gitattributes` file.

Comment (dnl-blkv) [inline .gitattributes]: We've got rid of `composer.lock` in https://github.com/bunq/sdk_php/pull/26 after all. Could you please drop it here as well? 

Also, don't we want to `export-ignore` the `CONTRIBUTING.md` and `/test/`, just like shown here: https://blog.madewithlove.be/post/gitattributes/?

Comment (cafferata) [inline .gitattributes]: Updated, also with a view to GrumPHPAlso with a view to GrumPHP (https://github.com/bunq/sdk_php/pull/24/).

**Pull request merged**